ibazel/main.go

```diff
@@ -57,7 +62,7 @@ target, run, build, or test the specified targets.
 
 Usage:
 
-ibazel [flags] build|test|run targets...
+ibazel build|test|run [flags] targets...
```

Comment:
could you make this be 

```
ibazel [startup_flags] build|test|run [flags] targets...
```

unless I'm incorrect in thinking it'll work like that

Reply:
Yes, I expected that to work as well, however:
```
$ ./ibazel --bazelrc=bazelrc.test build //...
flag provided but not defined: -bazelrc
iBazel - Version v0.10.3-dirty

A file watcher for Bazel. Whenever a source file used in a specified
target, run, build, or test the specified targets.

Usage:

ibazel [flags] build|test|run targets...

Example:
...
```

I suspect it has to do something how `flags.Parse()` works.

Any suggestions?

Reply:
I think aiming to keep the same CLI invocation is noble, but it looks like I need to revisit the logic there to make it actually behave that way. Could I have you file an issue requesting this and I'll add it to my backlog.

Thanks!

Reply:
I just open an issue regarding this: #283.